integration-tests/src/test/java/org/apache/fineract/integrationtests/LoanTransactionChargebackTest.java

```diff
@@ -66,7 +67,8 @@ public void setup() {
         this.requestSpec = new RequestSpecBuilder().setContentType(ContentType.JSON).build();
         this.requestSpec.header("Authorization", "Basic " + Utils.loginIntoServerAndGetBase64EncodedAuthenticationKey());
         this.responseSpec = new ResponseSpecBuilder().expectStatusCode(200).build();
-        this.responseSpecError = new ResponseSpecBuilder().expectStatusCode(503).build();
+        this.responseSpecErr400 = new ResponseSpecBuilder().expectStatusCode(400).build();
```

Comment:
this.responseSpecErr400 is not used anywhere. Would you mind to undo these changes if they are not necessary?